OpenIPSL/PowerPlantDataSets/Anderson/Combustion/AndersonCT1G.mo

```diff
@@ -1,4 +1,4 @@
-within OpenIPSL.DataSets.Anderson.Combustion;
+within OpenIPSL.PowerPlantDataSets.Anderson.Combustion;
```

Comment:
This may not be the right way to go. If you look at the top-level structure of OpenIPSL the most natural place for this would be inside `NonElectrical`. Then the question is what the name of the package itself should be. This also depends on what plans for future datasets are already in the making. Maybe @lvanfretti has some ideas here. We should not clutter up the namespace with long, convoluted, all too specific package names. 

What about: `OpenIPSL.NonElectrical.Data.PowerPlant`?

Reply:
@dietmarw, assuming we would include other types of data, which it is not clear if it would be a priority, my suggestion would be then to organize it as follows:
`OpenIPSL.Data.PowerPlant`

That would allow to include other data set arrangments, for example, groups of single phase transformers could be under:
`OpenIPSL.Data.Transformers`
Or for different "typical" power lines under:
`OpenIPSL.Data.TransmissionLines`
And so on...

I prefer have the `Data` subpackage on the top of the hierarchy, and not under, `NonElectrical` because it is not really describing behavior, just parameter values.

Reply:
Ok. I made the suggested changes.